src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -3,12 +3,15 @@
 
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
 using System.Text;
 
 namespace System.IO
 {
-    public static partial class Path
+    public static unsafe partial class Path
     {
+        private static readonly delegate* unmanaged<int, char*, uint> s_GetTempPathWFunc = GetGetTempPathWFunc();
```

Comment:
This initialization will get triggered any time other statics on Path get used. It is unnecessary and not trimmer friendly. We should initialize the pointer only when somebody actually calls GetTempPath.

Reply:
Done.